### DISCUSSION TITLE
Is array contains supported?

### DISCUSSION THREAD

**Discussion by joshuaavalon** [sinclairzx81/typebox#438] [Q&A] [answered]
https://json-schema.org/understanding-json-schema/reference/array.html#contains

`contains` is keyword of draft 6 but it does not seem to be exists on `ArrayOptions`. Is array contains supported?

**Comment by sinclairzx81** [comment, marked as answer]
@joshuaavalon Hi,

TypeBox doesn't expose the `contains` keyword (currently) as it's not supported in the type checkers, however you are still free to specify the `contains` keyword (as well as `minContains` and `maxContains`) on the `ArrayOptions`. 

The following is fine if using Ajv.

```typescript
// Expect an array of numbers, one or more should be equal to 1.
const T = Type.Array(Type.Number(), {
  contains: Type.Literal(1),
})
```
Maybe I should implement type checker support for this keyword? (There's some considering going into implement additional keywords that mean little to TS, but can be useful on the validation side, other examples include `if/then/else`, `dependentRequired` etc)

Hope this helps
S
